doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -213,6 +213,9 @@ nRF5340 Audio
 * Added:
 
   * Experimental support for Audio on the nRF5340 DK, with LED state indications and button controls.
+  * Support for stereo in broadcast_sink.
+    The :ref:`broadcast sink <nrf53_audio_broadcast_sink_app>` can now receive audio from two BISes and play it on the left and right channels of the audio output, given that the correct configurations are enabled.
```

Comment:
Should add that that the stereo out will be on I2S, whilst the audio DK codec is still mono. Can also refer to the commands to switch and mix L and R in the HW codec.